src/test/java/org/htmlunit/html/HtmlImageTest.java

```diff
@@ -318,4 +318,44 @@ public void clickWithCoordinates() throws Exception {
         img.click(2, 7);
         assertEquals(getExpectedAlerts()[0] + getExpectedAlerts()[1], page.getTitleText());
     }
+
+    /**
+     * @throws Exception on test failure
+     */
+    @Test
+    @Alerts({"begin-end-onload-", "onclick-begin-end-onload-", "onmousedown-onmouseup-begin-end-onclick-onload-"})
+    public void onload() throws Exception {
+        final String html = ""
+                + "<html>\n"
+                + "<head>\n"
+                + "<script>\n"
+                + "  function log(msg) { window.document.title += msg + '-';}\n"
+                + "  function loadImage() {\n"
```

Comment:
loadImage() is a bit confusing because this is the actual test rather than a method needed for the test...
```suggestion
                + "  function test() {\n"
```